.isort.cfg

```diff
@@ -0,0 +1,13 @@
+[settings]
+profile=black
+
+sections=FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+# These submodules are all added to the PYTHONPATH by Airflow, so we need to add them individually here.
+known_first_party=common,data_refresh,database,maintenance,oauth2,providers,retired
+
+lines_after_imports=2
+multi_line_output=3
+include_trailing_comma=true
+force_grid_wrap=0
+ensure_newline_before_comments=true
+use_parentheses=true
```

Comment:
I tested this locally and I think all of these are defaults or are defined in our `.pre-commit-config.yaml`, so we can remove them!

```suggestion
```